build.xml

```diff
@@ -6,6 +6,7 @@
   <property file="build.properties"/>
   <property name="compile.java.version" value="1.6"/>
   <property name="compile.java.bootclasspath" value="${build.bootclasspath}"/>
+  <property name="compile.java.newerClassname" value="java.lang.AutoCloseable"/>
```

Comment:
I don't understand what AutoCloseable has to do with detecting JDK versions. Please annotate, as this looks to be using indirect properties of some aspect rather than something clearly related to JDK versions.

Reply:
It's a class in a newer version of the JDK than 1.6, given two lines above.